### ISSUE TITLE
Ideas for coverage badge - Adding codecov.io ?

### ISSUE THREAD

**Issue by vaibhavmule** [Albert-Gao/modern-es6-npm-boilerplate#9]
Ideas for coverage badge?

- https://codecov.io
- https://coveralls.io

**Comment by Albert-Gao** [comment]
Good idea, I personally use coveralls.io, but seems codecov.io is more appealing. Should give it a try.

Another thing, I think it's more of a tutorial, rather than setup, things like coveralls.io, most of things happens at their website, the only file sits in your repo, is a file which contains your personal key, which shouldn't be sent to the server :)

But yes, I think we should add one, at least a procedure on teaching people how to do that.
But we will make it as easy as possible. 

So, if codecov.io could provide more integrated experiences, should use it instead 🗡 

I think we just need to try codecov.io first, what do you think? :)

And, furthermore, thanks for your contribution to the repo!

**Comment by vaibhavmule** [comment]
Yeah, same feeling here. 

Let's try codecov.io and see how it goes.
